cluster/gce/gci/configure-helper.sh

```diff
@@ -25,6 +25,27 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+function convert-params-array-to-yaml {
+  # A helper function to convert the manifest args from a string to a yaml list
```

Comment:
Why?

Reply:
To get the yaml to work you need special indentation. Honestly I think this should be done in a templating language. This is the poor equivalent. Wanted to keep the array of env values consistent and then adapt them based on how we want to output them (yaml vs manifest)

Reply:
yaml is a superset of json. You can use json syntax if you wish. It's not clear why you don't want to use the command syntax?